pipeline_steps/common.groovy

```diff
@@ -30,13 +30,6 @@ def void create_workspace_venv(){
     # UG-613 change TMPDIR to directory with more space
     export TMPDIR="/var/lib/jenkins/tmp"
 
-    # UG-612 Install pip, setuptools, and wheel on the host
```

Comment:
OK, in that case I'd suggest that https://github.com/rcbops/rpc-gating/blob/master/playbooks/setup_jenkins_slave.yml#L126 changes to be ``pip install pip setuptools wheel -c rpc-gating/constraints.txt``